CMakeLists.txt

```diff
@@ -203,6 +203,7 @@ rocm_enable_clang_tidy(
         -google-readability-*
         -google-runtime-int
         -google-runtime-references
+	-misc-include-cleaner
```

Comment:
You need to disable `cppcoreguidelines-missing-std-forward` and `modernize-type-traits`.